src/utils.cpp

```diff
@@ -922,9 +922,6 @@ void mudlog(const char *str, struct char_data *ch, int log, bool file)
         case LOG_ECONLOG:
           check_log = PRF_ECONLOG;
           break;
-        case LOG_RADLOG:
```

Comment:
Please add back in the radlog here

Reply:
Wow. I have no idea how that got removed in the first place as I didn't touch that function at all. Nor had I any reason to... Either I was coding half asleep or the cat jumped on ctrl-x or something..rofl. Really this is crazy.
https://static1.srcdn.com/wordpress/wp-content/uploads/2020/11/tim-and-eric-mind-blown.jpg